db/seeds.rb

```diff
@@ -33,7 +33,7 @@
 roles = %i[trusted chatroom_beta_tester workshop_pass]
 
 num_users.times do |i|
-  name = "#{Faker::Name.name} #{Faker::Lorem.word.titleize}"
+  name = "#{Faker::Name.name} #{Faker::Lorem.word.titleize}"[0..29]
```

Comment:
```suggestion
  name = Faker::Name.unique.name
```
I actually would suggest keeping it simple here, then you wouldn't need to do anything down in the email area. I was able to create 10,000 usernames.